File: Projects/TransportathonHackathon.API/TransportathonHackathon.Domain/Entities/DriverLicense.cs
using Core.Persistence.Entities;

namespace TransportathonHackathon.Domain.Entities
{
    public class DriverLicense : Entity
    {
        public Guid DriverId { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Classes { get; set; }
        public bool IsNew { get; set; }
        public DateTime LicenseGetDate { get; set; }

        public virtual Driver Driver { get; set; }
    }
}

File: Projects/TransportathonHackathon.Clients/angular-ui/src/app/models/domain-models/driverLicense.ts
import { BaseEntity } from '../../core/models/entity';
import { Driver } from './driver';

export interface DriverLicense extends BaseEntity {
  driverId: string;
  firstName: string;
  lastName: string;
  classes: string;
  isNew: boolean;
  licenseGetDate: string;
  driver: Driver;
}
